sources/packages/api/src/route-controllers/cra-integration/cra-integration.module.ts

```diff
@@ -0,0 +1,23 @@
+import { Module } from "@nestjs/common";
```

Comment:
I was trying to avoid adding more and more load into the app.module and trying to have it more organized.
I don't think that this module would be likely to be moved to the root. Do you?
In general I was trying to follow a bit what is described in nestjs docs.

https://docs.nestjs.com/modules
![image](https://user-images.githubusercontent.com/61259237/119052467-7d578080-b979-11eb-8074-099e86740e7d.png)

Reply:
The module was moved to root as per recommendation.